templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} manufacturer:</h1>
+  <form action="" method="post">
+  {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Create">
```

Comment:
The submit button value is hardcoded as 'Create'. It should dynamically change based on whether the form is for creating or updating a manufacturer.